packages/svelte/src/internal/client/reactivity/sources.js

```diff
@@ -145,9 +183,10 @@ export function mutate(source, value) {
  * @param {Source<V>} source
  * @param {V} value
  * @param {boolean} [should_proxy]
+ * @param {boolean} [needs_previous]
```

Comment:
`needs_previous` is always true in dev mode if `should_proxy` is true, so we don't need it as a separate argument

```suggestion
```